### ISSUE TITLE
[BUG] PlantUML preview seems broken

### ISSUE THREAD

**Issue by Hipska** [shd101wyy/vscode-markdown-preview-enhanced#1917]
**_Describe_ the bug**

When having a README file with an embedded PlantUML diagram, the diagram used to be rendered and displayed as an image in the preview. This seems not to be working anymore.

**Environment**
- OS: macOS Sonoma 14.2.1
- VSCode: 1.85.1
- Markdown Preview Enhanced: 0.8.0

**To Reproduce**
1. Create markdown file with embedded PlantUML diagram
2. Click on preview
4. See raw PNG bytes

**Expected behavior**
The rendered PNG should be displayed as image instead of raw bytes.

**Markdown file**
See screenshot..

**Screenshots**
<img width="1444" alt="image" src="https://github.com/shd101wyy/vscode-markdown-preview-enhanced/assets/228588/bb93c8bc-8d65-4439-b9ee-bc11c912183d">

**Additional context**
N/A

**Comment by EnochLiou** [comment]
A possible solution: 
1. [Optional?]Remove All customized settings of markdown-preview-enhanced in settings.json
2. uninstall MPE
3. turn off all VsCode instance.
4. reinstall MPE 
5. restart VsCode.

Then everything seems to work again in my case.

Details of my case:
After I changed some settings of Markdown-Preview-Enhanced(MPE), suddenly both @import puml file and embedded puml got similar error.

I am not for sure which config is the root cause. Tried several settings without luck, including:
remove kroki.io setting or make it empty, 
apply a working plantuml pickweb locally, 
make plantuml-server empty but give the new plantuml.jar absolute path,
...

At last, I tried reinstallation, then both @import and embedded puml work again.

Root cause is still not being identified, though, my working settings for MPE ver. 0.8.13 are listed below as a clue for further investigation:

```json
    "plantuml.jar": "C:/FreeApps/plantuml/plantuml.jar",
    "plantuml.render": "PlantUMLServer",
    "plantuml.server": "http://localhost:18080/svg/",
    "plantuml.exportFormat": "svg",
    "markdown-preview-enhanced.alwaysShowBacklinksInPreview": true,
    "markdown-preview-enhanced.plantumlJarPath": "C:/FreeApps/plantuml/plantuml.jar",
    "plantuml.urlFormat": "svg",
    "markdown-preview-enhanced.krokiServer": "",
```
